voice/bxml/verbs/speakSentence.md

```diff
@@ -40,22 +40,106 @@ To choose a specific voice by name, use the `voice` attribute.
 
 ### SSML Tags Supported:
 
-The table below lists the SSML tags that Bandwidth supports.  Full details about the tags can be found at https://www.w3.org/TR/2010/REC-speech-synthesis11-20100907/
-TODO: put descriptions here?
-
-| **tag**      | **description** |
-|:-------------|:----------------|
-| `<break>`    |                 |
-| `<emphasis>` |                 |
-| `<lang>`     |                 |
-| `<mark>`     |                 |
-| `<p>`        |                 |
-| `<phoneme>`  |                 |
-| `<prosody>`  |                 |
-| `<s>`        |                 |
-| `<say-as>`   |                 |
-| `<sub>`      |                 |
-| `<w>`        |                 |
+List of supported SSML tags.
+
+Full details about SSML tags can be found at:
+https://www.w3.org/TR/2010/REC-speech-synthesis11-20100907/
+
+#### `<break>`
+Adds a pause to the speech. You can specify the duration of the pause by using either the `strength` or the `time` attributes.
+
+Attributes:
+ * `strength`: (optional) accepted values are: `none`, `x-weak`, `weak`, `medium` (default), `strong` or `x-strong`
+ * `time`: (optional) the duration of the pause in second or milliseconds (e.g. `1s` or `1000ms`) with a maximum value of 10 seconds
+
+#### `<emphasis>`
+The contained text is spoken with emphasis.
+
+Attributes:
+ * `level`: (optional) defines the strength of emphasis to be applied, accepted values are: `strong`, `moderate` or `reduced`
+
+#### `<lang>`
+Specifies the natural language of the content.
+
+Attributes:
+ * `xml:lang`: specifies the language, accepted values are:
+   * `da-DK`: Danish
+   * `nl-NL`: Dutch
+   * `en-AU`: English, Australian
+   * `en-GB`: English, British
+   * `en-IN`: English, Indian
+   * `en-US`: English, US
+   * `fr-FR`: French
+   * `fr-CA`: French, Canadian
+   * `hi-IN`: Hindi
+   * `de-DE`: German
+   * `is-IS`: Icelandic
+   * `it-IT`: Italian
+   * `ja-JP`: Japanese
+   * `ko-KR`: Korean
+   * `nb-NO`: Norwegian
+   * `pl-PL`: Polish
+   * `pt-BR`: Portuguese, Brazilian
+   * `pt-PT`: Portuguese, European
+   * `ro-RO`: Romanian
+   * `ru-RU`: Russian
+   * `es-ES`: Spanish, European
+   * `es-MX`: Spanish, Mexican
+   * `es-US`: Spanish, US
+   * `sv-SE`: Swedish
+   * `tr-TR`: Turkish
+   * `cy-GB`: Welsh
+
+#### `<p>`
+Adds a pause between paragraphs.
+
+#### `<phoneme>`
+Use phonetic pronunciation for specific text.
+
+Attributes:
+ * `alphabet`: (optional) specifies the phonemic/phonetic pronunciation alphabet, accepted values: `ipa` (International Phonetic Alphabet, see http://www.internationalphoneticalphabet.org)
+ * `ph`: phonetic symbols
```

Comment:
@mrmeyers99, here I've removed the `x-sampa` option from `alphabet` because that's something supported by Amazon but not by other vendors. `ipa` seems to be the only one supported by all (at least is the one mentioned in the ssml standard and supported by MS and IBM tts).

Should we remove the `alphabet` from here since we would only officially be supporting `ipa`?

Reply:
I mean, we can keep the support in the code, since that attribute is optional, but remove the option from the docs.

Reply:
hmm.. I'd like to avoid "hidden" features.  If we are going to remove it from the docs, let's also remove it from the code.

Reply:
@mrmeyers99 , just took a look at the code and remembered one thing.

Right now we use a generic parsing of the content of \<SpeakSentence\>\</SpeakSentence\> so we don't actually have classes mapping all of those ssml tags.

Currently we only check if the tags are the ones that we allow (`SpeakSentenceConverter.ALLOWED_SSML_MARKS = {"speak",            "break",            "emphasis",            "lang",            "p",            "phoneme",            "prosody",            "s",            say-as",            "sub"}`) but we don't have validations for the tags attributes. The only validation we do is to check that the content is valid with the ssml schema (`synthesis-nonamespace.xsd`).

So if we are going to check that the attributes follow what we put in the docs, I think we should open a jira to check all tags. Should I open that jira?